command/test-fixtures/hcl2_upgrade/unknown_builder/expected.pkr.hcl

```diff
@@ -0,0 +1,50 @@
+packer {
+  required_version = ">= 1.6.0"
+}
+
+variable "aws_access_key" {
+  type    = string
+  default = ""
+}
+
+variable "aws_region" {
+  type = string
+}
+
+variable "aws_secret_key" {
+  type    = string
+  default = ""
+}
+
+source "potatoes" "autogenerated_1" {
```

Comment:
I want this builder now! If it's building potatoes 🥔

Reply:
Hahaha ! 🥔🥔🥔🥔🥔